packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -66,7 +66,8 @@ if (__DEV__) {
   var currentDebugStack = null;
   var currentDebugElementStack = null;
   var setCurrentDebugStack = function(stack) {
-    currentDebugElementStack = stack[stack.length - 1].debugElementStack;
+    var frame: Frame = stack[stack.length - 1];
```

Comment:
since this entire chunk of code is inside a `if (__DEV__)` block I think we could safely say that `var frame: FrameDev = stack[stack.length - 1];`

Reply:
I guess yea.